packages/appstore/src/components/modals/real-wallets-upgrade/components/title-text/title-text.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import './title-text.scss';
+
+const TitleText = ({ children, className }: React.PropsWithChildren<{ className?: string }>) => {
```

Comment:
Not sure about this component, are you sure we can't use usual `Text` component? As I know we can pass classname inside of the Text and override all unnecessary styles. Just a question, you can ignore it.

Reply:
I agree with @sergei-deriv. Text component already has children and classname 🤔

Reply:
Please check the styling of TitleText 🙏 . The same style is used in multiple places across different components.

<img width="538" alt="Screenshot 2023-10-05 at 11 23 25 AM" src="https://github.com/binary-com/deriv-app/assets/125039206/75e5f2e3-65df-4f58-9e35-081ec87f81c8">